### ISSUE TITLE
extension needs firefox restart in order to work

### ISSUE THREAD

**Issue by maxwell** [copyninja/cubbies-firefox#2]
it seems to install and activate after installation, but it only worked for me after I restarted firefox.  Perhaps their is a setting in the FF manifest file to prompt the user to restart her browser?

**Comment by copyninja** [comment]
Yeah I've seen this but it doesn't require restart it requires a page reload. i.e if you have already open page in the browser where images are present you need to reload the page because the scripts written (original scripts from Chrome plugin) are loaded (page-mod package in addon-sdk) only when page is "ready". I did it as a hack I'll see how can I change this

**Comment by copyninja** [comment]
With the current commit it doesn't even require a page reload install the addon and just right click on the required image and select **_Save to Cubbi.es**_ So I hope this issue is resolved now. Please confirm
